src/main/kotlin/com/atlassian/performance/tools/awsinfrastructure/api/jira/StandaloneFormula.kt

```diff
@@ -249,6 +255,10 @@ class StandaloneFormula private constructor(
         val databaseDataLocation = setupDatabase.get()
         val node = time("start") { provisionedNode.start(emptyList()) }
 
+        if (!selfAccess.get()) {
```

Comment:
For future changes we might think of timeouts

Reply:
Indeed, there should be a timeout for every `Future.get` call in all of JPT, to avoid hanging.

Reply:
I will not implement this suggestion i this PR - I believe we should create an issue about it